### PR TITLE
Minor typo fix

### PR DESCRIPTION
**Pull request by KRicky13**
#### Summary
Fixes a minor doc typo under `Parameters`

#### Motivation
First PR and to improve readability of doc

#### Supporting details
N/A

#### Related issues
N/A

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/atob/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ var decodedData = atob(encodedData);
 ### Parameters
 
 - `encodedData`
-  - : A [binary string](/en-US/docs/Web/API/DOMString/Binary) contains an
+  - : A [binary string](/en-US/docs/Web/API/DOMString/Binary) containing
     base64 encoded data.
```
Comment (teoli2003) [inline files/en-us/web/api/atob/index.md:35]: I think a dash is needed there too.
```suggestion
    base64-encoded data.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Good catch. A small addition and we can land this.

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Comment by KRicky13** [comment]
Woohoo! Thanks all for reviewing and approving. How exciting!

**Pull request merged**